### ISSUE TITLE
[Request] A way to input current crafting level?

### ISSUE THREAD

**Issue by kazerniel** [xanthics/gw2craft#32]
It would be nice to be able to input a character's current crafting level, and just calculate the leftover levels needed for 400. As it is now it's more difficult to use the site if the character already has some crafting lvls, and starter recipes don't give xp anymore.

**Comment by SuperJugy** [comment]
I agree with this one. specially so in the higher level ones. one of my chars is 425 and using the guide will make me start from 400. but i cannot just pick the guide from 425 because from 425 onwards it is assuming that you made discovery parts from level 400, sooo, i cannot use the guide or i will end up with extra materials after the guide which means wasted gold, which was the whole point of the guide in the first place.
